cmd/ltassist/main.go

```diff
@@ -173,18 +165,30 @@ func validTypes() string {
 	return s
 }
 
-func readDefaultConfig(configType, defaultPath string) (control.Config, error) {
+func getDefaultConfig(configType string) (interface{}, error) {
+	var cfg interface{}
+	var err error
 	switch configType {
 	case "agent":
-		return loadtest.ReadConfig(defaultPath)
+		cfg = &loadtest.Config{}
+		err = defaults.Set(cfg)
```

Comment:
Any reason we cannot move `defaults.Set` outside the switch-case ?

Reply:
Good point 👍